kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/archetype-resources/src/main/java/DefaultWebSecurityConfig.java

```diff
@@ -0,0 +1,29 @@
+package ${package};
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+
+@Configuration("kieServerSecurity")
+@EnableWebSecurity
+public class DefaultWebSecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http
+        .csrf().disable()
+        .authorizeRequests()
+        .antMatchers("/*").authenticated()
```

Comment:
maybe have security only for /rest/* so the landing page does not require authentication, wdyt?